libs/web3/src/lib/use-ethereum-config.ts

```diff
@@ -32,6 +32,19 @@ export const NETWORK_PARAMS_QUERY = gql`
   }
 `;
 
+export const useNetworkParameter = (params: string[]) => {
```

Comment:
chore (blocking): API now allows for getting a single param so lets refactor and use that.

Reply:
updated. updated also in token app to use web3 lib. the methods were duplicated. hope that is fine.